letter-sender/src/main/java/com/christmas/letter/sender/service/LetterService.java

```diff
@@ -0,0 +1,49 @@
+package com.christmas.letter.sender.service;
+
+import com.christmas.letter.sender.exceptions.LetterValidator;
+import com.christmas.letter.sender.model.Letter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import software.amazon.awssdk.services.sns.SnsClient;
+import software.amazon.awssdk.services.sns.model.PublishRequest;
+
+@Service
+@RequiredArgsConstructor
+public class LetterService {
+
+  @NonNull
+  private final SnsClient snsClient;
+  @NonNull
+  private final LetterValidator letterValidator;
+  private final ObjectMapper objectMapper;
+
+  private final String snsTopicArn = "arn:aws:sns:us-east-1:000000000000:ChristmasLetterSns";
```

Comment:
For this to be a constant, it should also be static, and let's update the name to follow the constant naming conventions, capital case only and _ as separator.

Reply:
updated to private static final String Sns_Topic_Arn